jvm/src/main/kotlin/app/cash/trifle/Certificate.kt

```diff
@@ -86,13 +71,13 @@ data class Certificate internal constructor(
         if (certificateRequest.pkcs10Req.subject == x509Certificate.subject
           && certificateRequest.pkcs10Req.subjectPublicKeyInfo == x509Certificate.subjectPublicKeyInfo
         ) {
-          VerifyResult(SUCCESS)
+          return true
```

Comment:
just `true` here without the return. The return statement is lifted, so the return keyword here should not be used.